gc.iml

```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Every attempt I've made to remove this file breaks the project in Android Studio. We should keep it.